app/assets/scripts/components/dashboard/details-card.js

```diff
@@ -20,6 +22,19 @@ export default function DetailsCard({ measurements, date, coords }) {
               {formatThousands(measurements)}
             </HighlightText>
             <CardSubtitle className="card__subtitle">Measurements</CardSubtitle>
+
+            {sources &&
+              sources
+                .filter(s => s.readme)
+                .map(s => (
+                  <SmartLink
+                    key={s.id}
+                    to={`${config.api}/${s.readme}`}
+                    title="Suggest a new source"
```

Comment:
This title is misleading, maybe
```suggestion
                    title="Get technical details"
```
?